feedback/views.py

```diff
@@ -594,7 +602,7 @@ def get_queryset(self):
         self.feedback_filter = filter = FeedbackFilter(self.request.GET, queryset, course=course)
         return filter.qs
 
-    def get_context_data(self, **kwargs):
+    def get_context_data(self, **kwargs) -> dict[str, Any]:
```

Comment:
In this file, `typing.Dict` has been imported, but you used builtin `dict` for the type. We should probably use only one of them. Seems like since Python 3.9, `dict` can now be used instead of `typing.Dict`: https://stackoverflow.com/a/78657334

Reply:
Indeed, and to my understanding using the builtin `dict` rather than `typing.Dict` (and the builtin `list` instead of `typing.List`) is recommended, which is why I used the builtin. Since in this PR it's such a minor change, I didn't want to change all the others here. In [another PR](https://github.com/apluslms/mooc-jutut/pull/109/files#diff-57474d9eceffcf4c288548a5c3cea35bd56475183c8356e323972143d7c61149) which edits the `update_context_for_feedback` function (which contains most of the type hints of the file), I change all the typing.Dict in this file to dict and remove the excess imports.